ogr/services/github/service.py

```diff
@@ -178,3 +179,41 @@ def project_create(
     def get_pygithub_instance(self, namespace: str, repo: str) -> PyGithubInstance:
         token = self.authentication.get_token(namespace, repo)
         return PyGithubInstance(login_or_token=token)
+
+    def list_projects(
+        self,
+        namespace: str = None,
+        user: str = None,
+        search_patter: str = None,
+        language: str = None,
+    ) -> List[GitProject]:
+        g = self.github
```

Comment:
You don't need to save it to a variable. It's a lazy property that is created only when needed and `g` is not a good name for a variable. I would say we don't get anything from this variable.